### DISCUSSION TITLE
nanodefender issues

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uAssets#21110] [General]
let's keep this thread for discussing issues from nano since its repositories are archived

**Comment by ghost** [comment]
`genvideos.com` - parked - redirected to a domain which is blocked by EasyList

**Comment by ghajini** [comment]
just open a PR, ubo team will check

**Reply by ghost** [reply]
fair, but sometimes there may be more / different ads etc.

**Comment by ghost** [comment]
@mapx- any other site regarding `https://github.com/NanoMeow/QuickReports/issues/1850` ?

**Comment by ghost** [comment]
`https://github.com/NanoMeow/QuickReports/issues/1932`

does `https://positivewordsdictionary.com/` load on someone else's side?
